src/libraries/System.Runtime/tests/System.Runtime.Tests/System/DateTimeTests.cs

```diff
@@ -14,6 +14,55 @@ namespace System.Tests
 {
     public class DateTimeTests
     {
+        [Fact]
+        public static void ParseExact_GenitiveMonthNames()
+        {
+            // Create a German culture with explicitly defined genitive month names
+            var culture = new CultureInfo("de-DE");
+            culture.DateTimeFormat.AbbreviatedMonthGenitiveNames = new[] { "Jan.", "Feb.", "März", "Apr.", "Mai", "Juni", "Juli", "Aug.", "Sept.", "Okt.", "Nov.", "Dez.", "" };
+            culture.DateTimeFormat.MonthGenitiveNames = new[] { "Januar", "Februar", "März", "April", "Mai", "Juni", "Juli", "August", "September", "Oktober", "November", "Dezember", "" };
```

Comment:
>Dezember [](http://example.com/codeflow?start=172&length=8)

Why did you change that and removed the dot as I suggested? most likely the test will fail. 

Also, did you run the test before you commit the change?